code/__DEFINES/~~~splurt_defines/intents.dm

```diff
@@ -0,0 +1,5 @@
+//INTENT VOODOO SHIT, ABANDON ALL HOPE YE WHO ENTER.
```

Comment:
I would appreciate a more detailed comment about what this is and why it works. You should probably mention how `0`, `null`, and `""` are all falsey values, in that in an `if()` condition they are considered to be false—so as far as things that only care about combat mode care, they're all equivalent to just having combat mode off, while harm intent is equivalent to combat mode being on.